### PR TITLE
Add Docker-compose file for windows compatibility 

### LINKED ISSUE #829: Cannot see the local version page using docker in README
OS: window 10 using MINGW

I want to check the edited page running local server with using `./bin/dockerhub_run.sh`.

After I run `./bin/dockerhub_run.sh` and connect the server, I see the second image below. (this means that cannot connect to site)

This is my process about I described above.
image
image

Thanks in advance !

### PR DESCRIPTION
**Pull request by pourmand1376**
This file makes it easier for windows users to use docker. (Closes #829)

Previous to this commit, those who used Windows had to install Ubuntu inside windows (via WSL) and run our commands. Now they can run it by just typing `docker-compose up`. 

> The main problem was that `./bin/dockerhub_run.sh` command was written with `Bash` in mind and you had to change it a little bit to make it compatible with windows `Powershell`. We shouldn't have two scripts. This is why adding a `docker-compose.yml` file is necessary.

### CODE REVIEW

**Comment by pourmand1376** [comment]
This commit changes the command users should type if they want to use docker. I've understood that using `docker-compose` is pretty much standard and it works with all operating systems (specially Windows). 

> Note: we still keep previous commands for compatibility. 

With this commit, we can use docker pre-built image using:
```bash
docker-compose up
```

we can build a custom image with: 
```bash
docker-compose -f docker-local.yml build
```

and we can use locally built image using:
```bash
docker-compose -f docker-local.yml up
```

**Comment by pourmand1376** [comment]
I added two parameters `--verbose` and `--livereload`. I have to see if those parameters add any problems here.

**Comment by pourmand1376** [comment]
I have tested this commit in Windows, and it is working well. Any comments? @alshedivat @rohandebsarkar

**Review by rohandebsarkar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -139,12 +139,12 @@ $ cd <your-repo-name>
 Finally, run the following command that will pull a pre-built image from DockerHub and will run your website.
 
 ```bash
-$ ./bin/dockerhub_run.sh
+$ docker-compose up
```
Comment (alshedivat) [inline README.md:142]: does this mean the user has to install `docker-compose`? if yes, let's add "install docker compose" step to the list on lines 131-132 above.

**Review by alshedivat — APPROVED** [review]
LGTM. thanks for adding `docker-compose` support. please see a small question about documentation below. once addressed, we can go ahead and merge.

File: README.md (hunk 1 above)
Comment (pourmand1376) [reply README.md:142]: Yes. I also added `docker-compose` installation step.

**Pull request merged**